implementations/app-swift/src/__tests__/cases/000-sanity/output/Types.swift

```diff
@@ -0,0 +1,243 @@
+    var circular: CustomType?
+    var const: String?
+}
+
+public struct CustomMapValue: Codable {
+    var foo: String
+}
+
+public struct Else: Codable {
+    var else: String
+}
+
+
+// Objects END //
+
+// Enums START //
+
+public enum CustomEnum: String, Codable {
+    case STRING
+    case BYTES
+}
+
+public enum While: String, Codable {
+    case _for
+    case _in
+}
+
+
+// Enums END //
+
+// Imported objects START //
+
+public struct TestImportObject: Codable {
+    var object: TestImportAnotherObject
+    var optObject: TestImportAnotherObject?
+    var objectArray: Array<TestImportAnotherObject>
+    var optObjectArray: Array<TestImportAnotherObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+public struct TestImportAnotherObject: Codable {
+    var prop: String
+}
+
+
+// Imported objects END //
+
+// Imported envs START //
+
+public struct TestImportEnv: Codable {
+    var object: TestImportAnotherObject
+    var optObject: TestImportAnotherObject?
+    var objectArray: Array<TestImportAnotherObject>
+    var optObjectArray: Array<TestImportAnotherObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+// Imported envs END //
+
+// Imported enums START //
+
+public enum TestImportEnum: String, Codable {
+    case STRING
+    case BYTES
+}
+
+public enum TestImportEnumReturn: String, Codable {
+    case STRING
+    case BYTES
+}
+
+
+// Imported enums END //
+
+// Imported modules START //
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsImportedMethod: Codable {
+    var str: String
+    var optStr: String?
+    var u: UInt32
+    var optU: UInt32?
+    var uArrayArray: Array<Array<UInt32?>?>
+    var object: TestImportObject
+    var optObject: TestImportObject?
+    var objectArray: Array<TestImportObject>
+    var optObjectArray: Array<TestImportObject?>?
+    var en: TestImportEnum
+    var optEnum: TestImportEnum?
+    var enumArray: Array<TestImportEnum>
+    var optEnumArray: Array<TestImportEnum?>?
+}
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsAnotherMethod: Codable {
+    var arg: Array<String>
+}
+
+// URI: "testimport.uri.eth" //
+public struct TestImportArgsReturnsArrayOfEnums: Codable {
+    var arg: String
+}
+
+/* URI: "testimport.uri.eth" */
+class TestImport {
+    var client: PolywrapClient? = nil
+    var env: TestImportEnv? = nil
+    var uri: Uri? = nil
+
+    init(client: PolywrapClient? = nil, env: TestImportEnv? = nil, uri: Uri? = nil) {
+        self.client = client
+        self.env = env
+        self.uri = uri
+    }
+
+    func getDefaultClient() -> PolywrapClient {
+        if (self.client == nil) {
+            self.client = BuilderConfig().addSystemDefault().addWeb3Default().build()
+        }
+        return self.client!
```

Comment:
We should not use `!` because its equivalent to `unwrap()` of a result in rust; I would to propose doing something like
```swift
if let client = self.client {
  return client
} else {
  let newClient = BuilderConfig()...
  self.client = newClient
  return newClient
}
``` 
this is more idiomatic imo